.storybook/preview.js

```diff
@@ -28,6 +28,13 @@ const preview = {
 			},
 
 		},
+		backgrounds: {
+			values: [
+				{ name: 'default', value: '#FFFFFF' },
```

Comment:
The value properties here could reference css variables rather than using hex colours, e.g. 
`{ name: 'default', value: 'var(--qld-default-background)' }`

Reply:
Thanks Jordan - I've realised we don't need to explicitly set a background for quotes...ie it will inherit by default. I've removed